### ISSUE TITLE
Possible error Github example

### ISSUE THREAD

**Issue by goyox86** [tomhoule/graphql-client#138]
Hi people!

First of all I'm really excited about this project.

I have been trying to run the stock Github example you have here and get:

```shell
    Finished dev [unoptimized + debuginfo] target(s) in 0.17s                                                                                                                                 
     Running `/home/goyox86/Code/rust/graphql-client/target/debug/graphql_query_github_example goyox86/blog`
there are errors:
Error { message: "No operation named \"Query1\"", locations: None, path: None, extensions: None }
thread 'main' panicked at 'missing response data', libcore/option.rs:1010:5
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::print
             at libstd/sys_common/backtrace.rs:71
             at libstd/sys_common/backtrace.rs:59
   2: std::panicking::default_hook::{{closure}}
             at libstd/panicking.rs:211
   3: std::panicking::default_hook
             at libstd/panicking.rs:227
   4: std::panicking::rust_panic_with_hook
             at libstd/panicking.rs:477
   5: std::panicking::continue_panic_fmt
             at libstd/panicking.rs:391
   6: rust_begin_unwind
             at libstd/panicking.rs:326
   7: core::panicking::panic_fmt
             at libcore/panicking.rs:77
   8: core::option::expect_failed
             at libcore/option.rs:1010
   9: <core::option::Option<T>>::expect
             at libcore/option.rs:322
  10: graphql_query_github_example::main
             at examples/github/src/main.rs:87
  11: std::rt::lang_start::{{closure}}
             at libstd/rt.rs:74
  12: std::panicking::try::do_call
             at libstd/rt.rs:59
             at libstd/panicking.rs:310
  13: __rust_maybe_catch_panic
             at libpanic_unwind/lib.rs:102
  14: std::rt::lang_start_internal
             at libstd/panicking.rs:289
             at libstd/panic.rs:392
             at libstd/rt.rs:58
  15: std::rt::lang_start
             at libstd/rt.rs:74
  16: main
  17: __libc_start_main
  18: _start
```

Thanks in advance!

**Comment by goyox86** [comment]
I just updated it initially I copied a wrong stacktrace.

**Comment by tomhoule** [comment]
Fixed by #147, thanks @h-michael !

**Comment by goyox86** [comment]
Thanks!
